src/annotator/config/index.js

```diff
@@ -51,3 +96,23 @@ export default function configFrom(window_) {
     ),
   };
 }
+
+/**
+ * Return the configuration for a given application context.
+ *
+ * @param {'sidebar'|'notebook'|'annotator'|'all'} [appContext] - The name of the app.
+ */
+export default function getConfig(appContext = 'annotator', window_ = window) {
+  // Filter the config based on the application context as some config values
+  // may be inappropriate or erroneous for some applications.
+  let partialConfig = {};
```

Comment:
`partialConfig` and `filteredKeys` are never re-assigned (`partialConfig` is modified, but it still refers to the same object), so both of these bindings can use `const` rather than `let`.